service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/orders/ApproveDraftOrdersController.java

```diff
@@ -77,10 +77,10 @@ public AboutToStartOrSubmitCallbackResponse validateReviewDecision(@RequestBody
 
         List<String> errors = approveDraftOrdersService.validateDraftOrdersReviewDecision(caseData, data);
 
-        if (isEmpty(errors) && isNotEmpty(caseData.getAllOthers())) {
+        if (isEmpty(errors) && isNotEmpty(caseData.getAllActiveOthers())) {
```

Comment:
could store result of `caseData.getAllActiveOthers` as a variable instead recomputing it multiple times